[docs] Update docs for "EAS" and "Classic Services" (#14537)
expo/expo: docs/pages/guides/environment-variables.md

sidebar_title: Environment variables
---

> > Are you using [EAS Build](/build/introduction/)? Make sure to check out our [Secrets documentation](/build-reference/variables.md), which lines out the best ways to integrate values that are too sensitive to include in your source code and git repository.

Reviewer: ```suggestion
> Are you using [EAS Build](/build/introduction/)? The [documentation about build secrets](/build-reference/variables.md) explains how to work with sensitive values that you would not include in your source code and Git repository.
```
